chart/epinio-installer/questions.yml

```diff
@@ -52,12 +52,12 @@ questions:
   type: boolean
   group: "General settings"
 - variable: skipLinkerd
-  label: Skip Linkerd installation
```

Comment:
Epinio can run even if there is no Linkerd. Maybe the new label is confusing in that case?

Reply:
Yeah, it could be. But I think it's closer to how a user looking at this with minimal background would expect to see it. We can document that if they don't want a service mesh, to just say they already have one and certain future features fail gracefully.

Reply:
Resolving and adding an issue to spend some time thinking through the Rancher specific workflow assumptions.